### ISSUE TITLE
[ACCESSIBILITY] Image selector buttons not announced 

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#1492]
**Describe the accessibility issue**
When swiping through app using assistive technology, user is unable to hear any controls when Uploading an image. 
[WCAG 2.1.1 Keyboard (A)](https://www.w3.org/TR/WCAG21/#keyboard)
[WCAG 2.1.2 No Keyboard Trap (A)](https://www.w3.org/TR/WCAG21/#no-keyboard-trap)

**To Reproduce**
**Steps to reproduce the behavior:**

1. Go to 'Create Group.'
2. Click on 'Add cover image.'
3. Select "Choose from device"'
4. Select 'Gallery'
5. Choose an image to upload, select it

Expected behavior
Upon selecting image to select, as a screen reader user I expect to hear something announced when a click sound occurs. This means I am selecting something with my TalkBack gesture but nothing is being read to me like "Confirm Image" or "Cancel upload" so I have no clue what the first four options are, then it move to a word "Free" which I have no idea how that relates to an image upload. Then it reads some ratios which again I have no clue have any relation to my image upload. Only a sighted user could proceed from here. THis is a show stopper in that sense. 

Screenshots
![Screenshot_20220323-101025](https://user-images.githubusercontent.com/82892873/160187541-1af9b0b5-d37d-4ec2-af41-28a080ed482d.png)

Smartphone (please complete the following information):
iPhone XR- iOS 14.6, Pixel 4a- Android 12

Additional context

**Comment by TodorBachvarov** [comment]
This panel is part of the external library that we use (https://pub.dev/packages/image_cropping). We don’t have control over the Semantics/Accessibility settings for this panel.

**Comment by acovher2** [comment]
I think we need our own accessible uploader and will submit a feature request for this. 
Thank you,
Amanda
